### ISSUE TITLE
Ionic 4.3.0 - cli freezes if starting project using type-parameter

### ISSUE THREAD

**Issue by jase88** [ionic-team/ionic-cli#3722]
**Description:**
Ionic CLI freezes cmd.exe on Windows if running ```ionic start someName blank --type=angular```
 - CLI last visible message is "Preparing directory done!"
- Pressing CTRL+C doesn't work, need to force to close cmd
- prepared folder stays empty

**Steps to Reproduce:**
run ```ionic start someName blank --type=angular``` on a Windows machine

**Output:**
```
C:\Users\my.username>ionic start someName blank --type=angular --verbose
  ionic:lib Terminal info: { tty: true, ci: false, windows: false } +0ms
  ionic:lib CLI global options: { _: [ 'start', 'someName', 'blank' ], help: null, h: null, verbose: true, quiet: null, interactiv
e: true, color: true, confirm: null, json: null, type: 'angular', project: null, '--': [] } +5ms
  ionic Context: { binPath: 'C:\\Users\\my.username\\AppData\\Roaming\\npm\\node_modules\\ionic\\bin\\ionic', libPath: 'C:\\Users\
\my.username\\AppData\\Roaming\\npm\\node_modules\\ionic', execPath: 'C:\\Users\\my.username', version: '4.3.0' } +0ms
V Preparing directory .\someName - done!
```

**My `ionic info`:**
ionic info not possible because Ionic project can't be created

**Other Information:**
If running  ```ionic start someName blank``` without the ionic 4 parameter "type" and confirm with y to start a ionic 4 project, everything works as expected

**Comment by dwieeb** [comment]
@kerosin Another workaround appears to be adding the `--no-interactive` flag.

**Comment by Liam-Ryan** [comment]
This is also affecting ionic cordova prepare statements

**Comment by nathantaal** [comment]
As referenced to 16213 by @kerosin, installing NodeJS 10.9 is also a workaround

**Comment by dwieeb** [comment]
The fix for this is in CLI 4.3.1!
